app/scripts/controllers/metamask-notifications/utils/utils.ts

```diff
@@ -231,7 +229,7 @@ export function checkTriggersPresenceByGroup(
       TRIGGER_TYPES.LIDO_STAKE_COMPLETED,
       TRIGGER_TYPES.LIDO_WITHDRAWAL_REQUESTED,
       TRIGGER_TYPES.LIDO_WITHDRAWAL_COMPLETED,
-      TRIGGER_TYPES.LIDO_STAKE_READY_TO_BE_WITHDRAWN,
+      // TRIGGER_TYPES.LIDO_STAKE_READY_TO_BE_WITHDRAWN,
```

Comment:
Should be removed.

Reply:
Yes, you’re right. I uncommented the line because it is actually necessary :)